pkgs/test/nixpkgs-check-by-name/README.md

```diff
@@ -8,16 +8,10 @@ This is part of the implementation of [RFC 140](https://github.com/NixOS/rfcs/pu
 
 This API may be changed over time if the CI workflow making use of it is adjusted to deal with the change appropriately.
 
-- Command line: `nixpkgs-check-by-name <NIXPKGS>`
+- Command line: `nixpkgs-check-by-name [--base <BASE_NIXPKGS>] <NIXPKGS>`
 - Arguments:
+  - `<BASE_NIXPKGS>`: The path to the Nixpkgs to check against
```

Comment:
If not specified, ...

Reply:
Good point! Also had to fix the behavior in that case (though it doesn't matter much): https://github.com/NixOS/nixpkgs/pull/272395/commits/53b43ce0e322eb4fc8daaad8a5a597155d42379a